### PR TITLE
fix: 🐛 fix vulnerability (pillow)

### PR DESCRIPTION
**Pull request by severo**
see https://github.com/huggingface/datasets-server/security/dependabot/272

also update libcommon in front/admin

### CODE REVIEW

File: front/admin_ui/poetry.lock [hunk 1]
```diff
@@ -236,6 +236,46 @@ files = [
...
+[package.dependencies]
+botocore = ">=1.31.59,<1.32.0"
+jmespath = ">=0.7.1,<2.0.0"
+s3transfer = ">=0.7.0,<0.8.0"
+
+[package.extras]
+crt = ["botocore[crt] (>=1.21.0,<2.0a0)"]
+
+[[package]]
+name = "botocore"
```
Comment (albertvillanova) [inline front/admin_ui/poetry.lock:260]: I am wondering why `boto3` and `botocore` are added by upgrading pillow?

**Review by albertvillanova — COMMENTED** [review]
I would mention this is a follow-up of the PR:
- #1853

File: front/admin_ui/poetry.lock (hunk 1 above)
Comment (severo) [reply front/admin_ui/poetry.lock:260]: I run `poetry update libcommon` at the same time, because it was not up to date

Comment (albertvillanova) [reply front/admin_ui/poetry.lock:260]: And `s3transfer`...

Comment (albertvillanova) [reply front/admin_ui/poetry.lock:260]: I see... I think it is because of the indirect dependency on `libcommon`...

Comment (severo) [reply front/admin_ui/poetry.lock:260]: yes, it's what I mentioned above

> I run poetry update libcommon at the same time, because it was not up to date

**Review by albertvillanova — APPROVED** [review]

**Pull request merged**